nearcore/src/config.rs

```diff
@@ -296,11 +298,15 @@ pub struct Config {
     /// because the previous block isn't available. The witnesses wait in the pool untl the
     /// required block appears. This variable controls how many witnesses can be stored in the pool.
     pub orphan_state_witness_pool_size: usize,
-
     /// The number of the contracts kept loaded up for execution.
     ///
     /// Each loaded contract will increase the baseline memory use of the node appreciably.
     pub max_loaded_contracts: usize,
+    /// Maximum size (number of bytes) of state witnesses in the OrphanStateWitnessPool.
+    ///
+    /// We keep only orphan witnesses which are smaller than this size.
+    /// This limits the maximum memory usage of OrphanStateWitnessPool.
+    pub orphan_state_witness_max_size: u64,
```

Comment:
Assuming this is size in bytes, please use Bytesize. It's config compatible and you won't need to parse it down the line.

Reply:
Great, changed to Bytesize.